src/codegen/properties/attribute.py

```diff
@@ -276,13 +277,13 @@ def get_argument_types(self):
     
     def get_return_type(self):
         if self.is_enum and not self.is_list:
-            return 'class: {0}'.format(self.enum)
+            return ':class: {0}'.format(self.enum)
         elif self.is_object and not self.is_list:
-            return 'class: {0}'.format(self.object_type)
+            return ':class: {0}'.format(self.object_type)
         elif self.is_enum and self.is_list:
-            return 'class: list({0})'.format(self.enum)
+            return ':class: list({0})'.format(self.enum)
         elif self.is_object and self.is_list:
-            return 'class: list({0})'.format(self.object_type)
+            return ':class: list({0})'.format(self.object_type)
```

Comment:
These have an extra space after `:class:` and are missing the backticks around the type. See https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects

![image](https://user-images.githubusercontent.com/17914399/220212051-1cc179aa-e9ce-4151-9fe5-50013c3fd8a8.png)

Reply:
As per the latest update in scrapigen, this logic will be removed in the subsequent PRs.